### PR TITLE
Fix TFFA from crashing when a player leaves during the intro sequence

### PR DESCRIPTION
**Pull request by x3Karma**
mog.

### CODE REVIEW

**Review by JMM889901 — APPROVED** [review]
Would it not also be worth just doing isvalid on respawnastitan?

**Comment by x3Karma** [comment]
> Would it not also be worth just doing isvalid on respawnastitan?

you probably could put isvalid checks on respawnastitan or respawnaspilot as well.

**Review by GeckoEidechse — APPROVED** [review]
Bug and fix confirmed in testing.

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_tffa.gnut [hunk 1]
```diff
@@ -63,6 +63,9 @@ void function PlayerWatchesTFFAIntroIntermissionCam( entity player )
 
 	wait TFFAIntroLength
 	
+	if (!IsValid(player)) // if player leaves during the intro sequence
```
Comment (RoyalBlue1) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_tffa.gnut:66]: ``
```suggestion
	if ( !IsValid( player ) ) // if player leaves during the intro sequence
```

**Review by RoyalBlue1 — APPROVED** [review]

**Pull request merged**